examples/library/src/Server.hs

```diff
@@ -0,0 +1,107 @@
+{-# language DataKinds             #-}
+{-# language FlexibleContexts      #-}
+{-# language OverloadedStrings     #-}
+{-# language PartialTypeSignatures #-}
+{-# language ScopedTypeVariables   #-}
+{-# language TypeApplications      #-}
+{-# language TypeFamilies          #-}
+{-# language TypeOperators         #-}
+{-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
+
+module Main where
+
+import           Control.Exception
+import           Control.Monad.Except
+import           Control.Monad.IO.Class            (MonadIO (..))
+import           Control.Monad.Logger
+import           Data.Conduit
+import           Data.Maybe                        (fromJust)
+import qualified Data.Text                         as T
+import           Database.Persist
+import           Database.Persist.Sqlite
+import           Mu.Adapter.Persistent             (runDb)
+import           Mu.GraphQL.Quasi
+import           Mu.GraphQL.Server
+import           Mu.Schema
+import           Mu.Server
+import           Network.Wai.Handler.Warp          (run)
+import           Network.Wai.Middleware.AddHeaders (addHeaders)
+
+import           Schema
+
+main :: IO ()
+main = do
+  putStrLn "starting GraphQL server on port 8000"
+  let hm = addHeaders [
+             ("Access-Control-Allow-Origin", "*")
+           , ("Access-Control-Allow-Headers", "Content-Type")
+           ]
+  runStderrLoggingT $
+    withSqliteConn @(LoggingT IO) ":memory:" $ \conn -> do
+      runDb conn $ runMigration migrateAll
+      liftIO $ run 8000 $ hm $
+        graphQLApp (libraryServer conn)
+                   (Proxy @('Just "Query"))
+                   (Proxy @('Just "Mutation"))
+                   (Proxy @('Just "Subscription"))
+
+type ObjectMapping = '[
+    "Book"   ':-> Entity Book
+  , "Author" ':-> Entity Author
+  ]
+
+libraryServer :: forall i.
+                 SqlBackend
+              -> ServerT ObjectMapping i Library ServerErrorIO _
+libraryServer conn
```

Comment:
Since now we have a complete example on a separate folder, can we remove the `Example.hs` in the graphql package?